backbone/src/main/res/values-cs-rCZ/strings.xml

```diff
@@ -110,47 +111,46 @@
-    <string name="mtrl_picker_invalid_range" tools:override="true" tools:ignore="ResourceName">Neplatný rozsah.</string>
-    <string name="mtrl_picker_out_of_range" tools:override="true" tools:ignore="ResourceName">Mimo dosah: %1$s</string>
-    <string name="mtrl_picker_invalid_format" tools:override="true" tools:ignore="ResourceName">Nesprávný formát.</string>
-    <string name="mtrl_picker_invalid_format_use" tools:override="true" tools:ignore="ResourceName">Použití: %1$s</string>
-    <string name="mtrl_picker_invalid_format_example" tools:override="true" tools:ignore="ResourceName">Příklad: %1$s</string>
-    <string name="mtrl_picker_toggle_to_calendar_input_mode" tools:override="true" tools:ignore="ResourceName">Přepnutí do režimu zadávání kalendáře</string>
-    <string name="mtrl_picker_toggle_to_text_input_mode" tools:override="true" tools:ignore="ResourceName">Přepnutí do režimu zadávání textu</string>
-    <string name="mtrl_picker_a11y_prev_month" tools:override="true" tools:ignore="ResourceName">Změnit na předchozí měsíc</string>
-    <string name="mtrl_picker_a11y_next_month" tools:override="true" tools:ignore="ResourceName">Změnit na další měsíc</string>
-    <string name="mtrl_picker_toggle_to_year_selection" tools:override="true" tools:ignore="ResourceName">Klepnutím přepnete na výběr roku</string>
-    <string name="mtrl_picker_toggle_to_day_selection" tools:override="true" tools:ignore="ResourceName">Klepnutím přepnete na výběr dne</string>
-    <string name="mtrl_picker_day_of_week_column_header" tools:override="true" tools:ignore="ResourceName">Sloupec dní: %1$s</string>
-    <string name="mtrl_picker_announce_current_selection" tools:override="true" tools:ignore="ResourceName">Aktuální výběr: %1$s</string>
-    <string name="mtrl_picker_navigate_to_year_description" tools:override="true" tools:ignore="ResourceName">Přejděte na rok %1$s</string>
-    ​
+    <string name="mtrl_picker_range_header_title" tools:ignore="ResourceName" tools:override="true">Vyberte rozmezí datumů</string>
+    <string name="mtrl_picker_date_header_title" tools:ignore="ResourceName" tools:override="true">Vyberte datum</string>
+    <string name="mtrl_picker_range_header_unselected" tools:ignore="ResourceName" tools:override="true">Počáteční datum – Koncové datum</string>
+    <string name="mtrl_picker_range_header_only_start_selected" tools:ignore="ResourceName" tools:override="true">%1$s – Koncové datum</string>
+    <string name="mtrl_picker_range_header_only_end_selected" tools:ignore="ResourceName" tools:override="true">Počáteční datum – %1$s</string>
+    <string name="mtrl_picker_range_header_selected" tools:ignore="ResourceName" tools:override="true">%1$s – %2$s</string>
+    <string name="mtrl_picker_date_header_unselected" tools:ignore="ResourceName" tools:override="true">Zvolené datum</string>
+    <string name="mtrl_picker_date_header_selected" translatable="false" tools:ignore="ResourceName" tools:override="true">%1$s</string>
```

Comment:
I think this one and the one below, should not have `translatable="false"`. (Check that I'm not wrong before changing stuff).

Reply:
I removed it from all strings file except the default one